### PR TITLE
John password and logo

### PR DESCRIPTION
**Pull request by jjbutler52**
Requesting review of changes Ben and I did to the password security validation in loginViewModel.kt and strings.xml

### CODE REVIEW

**Review by SauterBenjamin — APPROVED** [review]
Looks Good

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -7,6 +7,9 @@
     <string name="action_sign_in_short">Sign in</string>
     <string name="welcome">"Welcome !"</string>
     <string name="invalid_username">Not a valid UWF email</string>
-    <string name="invalid_password">Password must be >5 characters</string>
+    <string name="invalid_password_length">Password must be >8 characters</string>
+    <string name="invalid_password_capitalization">Password must contain at least 1 upper and 1 lower case letter</string>
+    <string name="invalid_password_character">Password must be contain at least 1 special character</string>
```
Comment (jemramsey) [inline app/src/main/res/values/strings.xml:12]: teeny grammar error, delete "be" from the string output for lines 12 and 13.

**Review by jemramsey — APPROVED** [review]

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (jjbutler52) [reply app/src/main/res/values/strings.xml:12]: I will edit it tomorrow, good catch

**Comment by ChrisMoone** [comment]
Looks good to me

**Review by ChrisMoone — APPROVED** [review]
Looks good to me

**Pull request merged**